docs/getting-started/setting-up-lab-environment.md

```diff
@@ -163,7 +165,7 @@ az aks create \
 --name ${AKS_NAME} \
 --location ${LOCATION} \
 --tier standard \
---kubernetes-version 1.29 \
+--kubernetes-version 1.31 \
```

Comment:
Instead of bumping versions each time a new one is released, maybe we should use `az aks get-versions` command and get the current default version.